### DISCUSSION TITLE
Use "standard" map color theme as default

### DISCUSSION THREAD

**Discussion by jtguggedal** [hello-nrfcloud/web#912] [Ideas]
I suggest we use a "standard" map layout as default, so that users quickly can see where a device is located in a familiar map layout.

By "standard" I mean a theme similar to Bing, Google maps and the likes, where water is blue and there are in general more colors to easily differentiate between main roads, smaller roads, forest, mountains etc.

It would also be very nice if the map could have more labels on roads, major buildings etc. Currently it seems quite minimalistic.

**Comment by coderbyheart** [comment]
This is intentional but we can add a theme style switcher (like we have for dark/bright style).

**Comment by bjda** [comment]
Minimalist maps are often used when overlaying data, so that you don't get a too busy result with a lot of text and and colors competing for attention. That being said, I think we can try to use a colored map with some POI labels. Especially when showing just a position, the surrounding geography is just as important as the data. If the provider has a soft color scheme available, our data layer can still stand out with a higher saturation.
